app/controllers/checkins_controller.rb

```diff
@@ -128,5 +128,6 @@ def destroy
 
   def checkin_params
     params.require(:checkin).permit(:location_id, :team_id)
+    #params.require(:checkin).permit(:location_id, :team_id, :timestamp) # for when timestamp updating works
```

Comment:
It's new, I'm working on fixing the timestamps, or being able to update timestamps on the fly.